scripts/src/java/org/oppia/android/scripts/label/AccessibilityLabelCheck.kt

```diff
@@ -0,0 +1,156 @@
+package org.oppia.android.scripts.label
+
+import org.oppia.android.scripts.proto.AccessibilityLabelExemptions
+import org.w3c.dom.Node
+import org.w3c.dom.NodeList
+import java.io.File
+import java.io.FileInputStream
+import java.util.stream.Collectors
+import java.util.stream.IntStream
+import javax.xml.parsers.DocumentBuilderFactory
+
+/**
+ * Script for ensuring that all the Activities in the codebase are defined with accessibility
+ * labels.
+ *
+ * Usage:
+ *   bazel run //scripts:accessibility_label_check -- <path_to_directory_root> <path_to_app_level
+ *   _manifest_file>
+ *
+ * Arguments:
+ * - path_to_directory_root: directory path to the root of the Oppia Android repository.
+ * - path_to_app_level_manifest_file: path to the manifest file of app layer.
+ *
+ * Example:
+ *   bazel run //scripts:accessibility_label_check -- $(pwd) app/src/main/AndroidManifest.xml
+ */
+fun main(vararg args: String) {
+  val repoPath = "${args[0]}/"
+
+  val accessibilityLabelExemptiontextProto = "scripts/assets/accessibility_label_exemptions"
```

Comment:
Ending should be ``TextProto`` (as separate words in the identifier).

Ditto in the function below with the parameter of the same name.

Reply:
Done.